files/ko/learn/javascript/asynchronous/async_await/index.html

```diff
@@ -83,7 +92,7 @@ <h3 id="The_await_keyword">The await keyword</h3>
 
 <p>물론 위의 예시는 그다지 쓸모있진 않습니다. 다만 어떻게 구문을 작성해야 하는지는 잘 나타내줍니다. 이제 실제 사례를 살펴봅시다.</p>
 
-<h2 id="Rewriting_promise_code_with_asyncawait">Rewriting promise code with async/await</h2>
+<h2 id="Rewriting_promise_code_with_asyncawait">다시 쓰는 promise code 그리고 async/await</h2>
```

Comment:
`Promise` 코드와 `async` 코드를 같이 적는 예제여서 아래와 같이 바꾸는게 적절할거 같습니다!
```suggestion
<h2 id="Rewriting_promise_code_with_asyncawait">async/await와 함께 다시 쓰는 promise code</h2>
```